simpleInstall/k2config_pgrun.json

```diff
@@ -3,9 +3,23 @@
         "host": "172.17.0.1",
         "port": 30000
     },
+    "k2.txn_op_timeout_ms": 3600000,
+    "k2.sync_finalize_txn": true,
     "create_collections": {
         "PG_DEFAULT_CLUSTER": {
             "range_ends": [""]
+        },
+        "template1": {
+            "range_ends": [""],
+            "endpoints": ["tcp+k2rpc://127.0.0.1:10001"]
```

Comment:
The endpoints here don't actually do anything anymore, the cpo will select the endpoints on its own

Reply:
good catch. removed.